grader/ajax_key.py

```diff
@@ -0,0 +1,6 @@
+"""
+Automatically generated file.
+This needs to be unique and SECRET. It is also installation specific.
+This file should be included from settings.py
+"""
+AJAX_KEY = 'rr)4qe+#xe7-$lbeaif=+w2ei^c&vc!3rq^*=hw=jbyiekri21'
```

Comment:
oh. these files can be added to `.gitignore`, in a separate commit. Again, related to local testing.